proxy/api/src/identity.rs

```diff
@@ -28,14 +38,22 @@ impl From<(coco::PeerId, coco::Person)> for Identity {
     fn from((peer_id, user): (coco::PeerId, coco::Person)) -> Self {
         let urn = user.urn();
         let handle = user.subject().name.to_string();
+        // Silently ignore the extension payload if malformed, the identity itself is still valid
```

Comment:
I have a feeling that silently ignoring will lead us to wonder why some error happened down the line :D Can we maybe log as a warning?

Reply:
Fixed, added logging